src/Okta.Sdk.IntegrationTests/AuthorizationServersScenarios.cs

```diff
@@ -0,0 +1,603 @@
+                    await testClient.AuthorizationServers.GetAuthorizationServerAsync(createdAuthorizationServer.Id);
+                retrievedDeactivatedAuthorizationServer.Status.Should().Be("INACTIVE");
+            }
+            finally
+            {
+                await testClient.AuthorizationServers.DeactivateAuthorizationServerAsync(createdAuthorizationServer.Id);
+                await testClient.AuthorizationServers.DeleteAuthorizationServerAsync(createdAuthorizationServer.Id);
+            }
+        }
+
+        [Fact(Skip = "Api validation failed: com.saasure.framework.exception.KeyStoreLimitExceededException (400, E0000001): Unable to create new authorization server. You cannot create authorization servers because signing keys could not be generated. To fix this issue, contact support.")]
+        public async Task ActivateAuthorizationServer()
+        {
+            var testClient = TestClient.Create();
+            var testAuthorizationServerName = $"{SdkPrefix}:{nameof(DeactivateAuthorizationServer)}:Test Authorization Server ({TestClient.RandomString(6)})";
+
+            var testAuthorizationServer = new AuthorizationServer
+            {
+                Name = testAuthorizationServerName,
+                Description = "Test Authorization Server",
+                Audiences = new string[] { "api://default" },
+            };
+
+            var createdAuthorizationServer = await testClient.AuthorizationServers.CreateAuthorizationServerAsync(testAuthorizationServer);
+            try
+            {
+                createdAuthorizationServer.Should().NotBeNull();
+                createdAuthorizationServer.Status.Should().Be("ACTIVE");
+
+                await testClient.AuthorizationServers.DeactivateAuthorizationServerAsync(createdAuthorizationServer.Id);
+                await testClient.AuthorizationServers.ActivateAuthorizationServerAsync(createdAuthorizationServer.Id);
+
+                var retrievedDeactivatedAuthorizationServer = await testClient.AuthorizationServers.GetAuthorizationServerAsync(createdAuthorizationServer.Id);
+                retrievedDeactivatedAuthorizationServer.Status.Should().Be("ACTIVE");
+            }
+            finally
+            {
+                await testClient.AuthorizationServers.DeactivateAuthorizationServerAsync(createdAuthorizationServer.Id);
+                await testClient.AuthorizationServers.DeleteAuthorizationServerAsync(createdAuthorizationServer.Id);
+            }
+        }
+
+        [Fact]
+        public async Task ListAuthorizationServerPolicies()
+        {
+            var testClient = TestClient.Create();
+            var defaultAuthorizationServer = await testClient.AuthorizationServers.ListAuthorizationServers().FirstOrDefaultAsync(authorizationServer => (bool)authorizationServer?.Name?.Equals("default"));
+
+            var policies = await testClient.AuthorizationServers.ListAuthorizationServerPolicies(defaultAuthorizationServer.Id).ToListAsync();
```

Comment:
Is it possible to create our own test policy here? I know it's not possible to create AS today, but when the key storage issue is fixed. I wouldn't rely on default policies here if we don't have a way to create our own, I would create a UT instead.

Reply:
I'll update.